wordpress-mobile/WordPress-iOS
# Issue/fix rotation of photos

Fixes #12703 

This PR bings fixes for two issues:

 - Images that were captured inside the app using a non-standard orientation of the device (ex: upside down) were being saved incorrectly in the camera roll
- Images that have a non-standard orientation, capture inside the app or in the camera app, when updated to a WP.org site that was is bellow version 5.3 will be show incorrectly on the web.

For the first issue we are updating the WPMediaPicker version that has a fix to save the images correctly when taken inside the app

For the second issue we are now applying a transform on all images uploaded from the app to make sure that they all have orientation up independently of the size setting. 

To test:
 - Capture a photo inside the app with the device upside down
 - Make sure the image is recorded with the correct orientation and display properly inside the media library
 - Use that photo on a post and upload it to a WP.org site that is bellow version 5.3 ( it cannot be connected with Jetpack or else the Photon will fix/hide the issue)
- Upload/Publish the Post
- See the post on the web, using Safari, to see if photo shows with the correct orientation.

Update release notes:
